### ISSUE TITLE
Download a course that I want to complete but not completed yet

### ISSUE THREAD

**Issue by Yousuf28** [TRoboto/datacamp-downloader#8]
I have an active subscription to Datacamp. Is there any way to download a course that I have not completed yet?

**Comment by MuhammadAlBarham** [comment]
Unfortunately, you cannot. The tool downloads only your completed track(s) or course(s).

**Comment by Yousuf28** [comment]
Thanks for the response. I have downloaded courses that I completed and it works great. But I think it would not be hard to work around and download courses that not completed yet.  the get_completed_courses function saves all the completed courses link. If I can provide any course link here it should download course materials. I am not sure what changes I need to do in other places. I would appreciate it if anyone works on that or give some suggestions.
